.github/workflows/go.yml

```diff
@@ -66,7 +85,14 @@ jobs:
         go-version-file: "go.mod"
 
     - name: Run Race Tests
-      run: go test -race ./...
+      run: go test -race ./... -coverpkg=./... -coverprofile=race_coverage.txt
```

Comment:
Is it possible to run the race tests a number of times?

Reply:
It might be worth having something like 10 rounds for race tests; testing locally, I do this to make sure any race issues have been fixed, but 100 iterations might be overkill for CI 😅 

```bash
for i in {1..100}; do go clean -testcache; go test ./... -race; done
```